snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsModifiers/CS/csrefKeywordsModifiers.cs

```diff
@@ -228,8 +228,7 @@ public class Publisher
         protected virtual void RaiseSampleEvent()
         {
             // Raise the event by using the () operator.
```

Comment:
I like this change. We should change the comment to match the new code: 

```suggestion
            // Raise the event in a thread-safe manner using the ?. operator.
```

Reply:
You're right. How did I miss that XD